client/my-sites/media-library/content.jsx

```diff
@@ -118,7 +120,27 @@ export default React.createClass( {
 				args: occurrences.length
 			};
 
+			if ( this.props.site ) {
+				onDismiss = MediaActions.clearValidationErrorsByType.bind( null, this.props.site.ID, errorType );
+			}
+
 			switch ( errorType ) {
+				case MediaValidationErrors.FILE_TYPE_NOT_IN_PLAN:
+					return (
+						<Notice status="is-warning" onDismissClick={ onDismiss }>
+							{
+								this.translate(
+									'Your site does not support video files. {{a}}Upgrade to Premium for video support{{/a}}.',
+									merge( i18nOptions, {
+										components: {
+											a: <a target="_blank" href={ '/plans/' + this.props.site.slug } />,
```

Comment:
Because this notice has "Dismiss" button also and Notice action looks terrible when there's dismiss. We should really update the notice compoennt